### PR TITLE
Update documentation for repository contributors

### PR DESCRIPTION
**Pull request by rin-skylight**
# DOCUMENTATION PULL REQUEST

## Related Issue

- Why is this being done? Link to issue, or a few sentences describing why this PR exists

## Changes Proposed

- Fill out detailed `CONTRIBUTING.md`, adapting the file currently used by ReportStream to use more stringent standards.
- Add a `code-of-conduct.md`, copying that used in ReportStream verbatim.

## Additional Information

These files are open to further modifications from each SimpleReport team, as well as any other input that may be desired. They shape our future stance on collaboration, so we should be thoughtful and deliberate with our choices.

Human-friendly link for `CONTRIBUTING`: https://github.com/CDCgov/prime-simplereport/blob/rin/contributing/CONTRIBUTING.md
Human-friendly link for `code-of-conduct`: https://github.com/CDCgov/prime-simplereport/blob/rin/contributing/code-of-conduct.md

### CODE REVIEW

**Comment by rin-skylight** [comment]
@emmastephenson, @zdeveloper, I wanted to open this to review from both of you before it went to the team at large, since these standards are up to you to enforce as technical leads. Please let me know your thoughts or suggested changes; we can open it to the team at large once you've given it a first pass.

**Comment by emmastephenson** [comment]
@rin-skylight it looks great to me! Thanks for pulling this together.

**Review by emmastephenson — DISMISSED** [review]

**Comment by rin-skylight** [comment]
Opening to the team for a final review.

**Comment by mehansen** [comment]
Do we have a new content person now that Rebecca's gone who should look at this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,46 +1,87 @@
-# Contributing to the PRIME Data Input Application
+# Welcome!
+Thank you for your interest in contributing to SimpleReport, part of the CDC's PRIME initiative. SimpleReport is an open-source application leveraged by organizations within the United States for collecting and reporting test results for various supported diseases. We're glad you are interested in contributing!
+There are many ways to contribute, including writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests, and writing code for PRIME itself.
 
-## Branch Management
+Before contributing, we encourage you to also read or [LICENSE](LICENSE.md),
```
Comment (mehansen) [inline CONTRIBUTING.md:5]: looks like this should be "our" not "or"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,46 +1,87 @@
...
-without an extremely compelling reason.
\ No newline at end of file
+### Fork and clone the repository
+
+You will need to fork the main code or documentation repository and clone it to your local machine. See
+[github help page](https://help.github.com/articles/fork-a-repo) for help. 
+
+Create a branch for your work. 
+
+If a you need to base your work on someone elses branch, talk to the branch owner and work something out.
```
Comment (mehansen) [inline CONTRIBUTING.md:48]: someone else's

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,46 +1,87 @@
...
+
+Once your changes and tests are ready to submit for review:
+
+1. **Test your changes**
+
+    Run the full local test suite to make sure that nothing is broken.
+
+2. **Rebase your changes**
+
+    Update your local repository with the most recent code from the principal repository, and rebase your branch on top of the latest master branch. We prefer your initial changes to be squashed into a single commit. Later, if we ask you to make changes, add the changes as separate commits.  This makes the changes easier to review.
```
Comment (mehansen) [inline CONTRIBUTING.md:65]: Should we use 'main' instead of 'master' for branch name? I think we're trying to move away from using the term master.

**Comment by mehansen** [comment]
What's the code of conduct for? Are we saying contributors must abide by this as well, including completing `security awareness and records management training annually`?

File: CONTRIBUTING.md (hunk 1 above)
Comment (rin-skylight) [reply CONTRIBUTING.md:5]: Great callout! I will push a fix.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rin-skylight) [reply CONTRIBUTING.md:48]: Thanks for the catch! That'll go in the fix commit, too.

File: CONTRIBUTING.md (hunk 3 above)
Comment (rin-skylight) [reply CONTRIBUTING.md:65]: Great callout on this, as well. I need to clean up the terminology and properly format the branch name...this was a copypasta, and I missed a lot of these smaller things. I really appreciate your attention to detail!

**Comment by rin-skylight** [comment]
> What's the code of conduct for? Are we saying contributors must abide by this as well, including completing `security awareness and records management training annually`?

Great question, @mehansen! The code of conduct file is designed to set a standard that everyone who contributes to this repository should strive to meet as they go about their work. This language was largely targeted towards HHS/CDC employees and contractors, since these groups and people do the bulk of the work on this. You have a great point about the security and records training; contributors outside of our organizations would not have access to these items, but it is assumed that outside contributors will not be in breach of guidelines as the required training never applied to them to begin with. (We can absolutely make this clearer if desired, though!)

**Comment by rin-skylight** [comment]
> Hi @mehansen! @rin-skylight @emmastephenson Are you open to a content pass on this? At a glance it looks great, but can take a closer look for plain language, etc.

If you have the time, this would be wonderful, @lauraykerr! We would love to make this as inclusive and easy-to-work-with as possible, while still setting down the proper guidelines.

**Review by mehansen — APPROVED** [review]

**Review by alismx — APPROVED** [review]

**Review by emmastephenson — APPROVED** [review]

**Comment by rin-skylight** [comment]
I'm going to go ahead and merge this, so we have a base on which to build. @lauraykerr , you're welcome to give this a look when you return from your break!

**Pull request merged**